kudu/pom.xml

```diff
@@ -51,5 +51,10 @@ LICENSE file.
       <artifactId>slf4j-log4j12</artifactId>
       <version>1.7.21</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>RELEASE</version>
```

Comment:
is 'RELEASE' some maven magic that I'm not aware of?

Reply:
It's very evil magic, and should be replaced by the use of a specific version.